src/widget/wtracktableview.cpp

```diff
@@ -1074,6 +1087,49 @@ void WTrackTableView::contextMenuEvent(QContextMenuEvent* event) {
             }
         }
         m_pMenu->addMenu(m_pBPMMenu);
+
+        // Track color menu only appears if at least one track is selected
+        if (indices.size()) {
+            // Get color of first selected track
+            int column = trackModel->fieldIndex(LIBRARYTABLE_COLOR);
+            QModelIndex index = indices.at(0).sibling(indices.at(0).row(), column);
+            mixxx::RgbColor::optional_t trackColor = std::nullopt;
+            if (!index.data().isNull()) {
+                trackColor = mixxx::RgbColor::optional_t(index.data().toUInt());
```

Comment:
`::optional(...)`, the static function instead of the typedef with the suffix `_t`